pkg/odo/cli/component/create.go

```diff
@@ -496,7 +496,7 @@ func (co *CreateOptions) Run() (err error) {
 			return errors.Wrapf(err, "failed to push the changes")
 		}
 	} else {
-		log.Infof("Please use `odo push` command to create the component with source deployed\n")
+		log.Infof("Please use `odo push` command to create the component with source deployed")
```

Comment:
Comparing function names with those in `fmt` package, my understanding is that `Printf` requires one to explicitly add a new-line character if they wish to while `Println` adds a new-line character by itself. Although the main difference, IIUC, is that `Printf` allows one to format the output while `Println` doesn't. In this case, IMHO, it is the behaviour of `log.Infof` that we should change instead of the change we are making here. WDYT?

Reply:
Our own logging adds newline out of convenience :)

Reply:
Yep, I understand that. But I think we could/should follow the convention of `Printf` vs. `Println`. Obviously I'm not going to hold this PR for that reason. :joy: